doc/source/point-in-time-recovery.rst

```diff
@@ -0,0 +1,84 @@
+.. _pitr:
+
+Point-in-Time Recovery
+********************************************************************************
+
+|PITR| is restoring a database up to a specific moment. |PITR| includes restoring the data from a backup snapshot and replaying all events that occurred to this data up to a specified moment from :term:`oplog slices <Oplog slice>`. |PITR| helps you prevent data loss during a disaster such as crashed database, accidental data deletion or drop of tables, unwanted update of multiple fields instead of a single one.
+
+|PITR| is available in |PBM| starting from v1.3.0. |PITR| is enabled via the ``pitr.enabled`` config option.
+
+.. code-block:: bash
+
+   $ pbm config --set pitr.enabled=true
+
+Incremental backups
+================================================================================
+
+When |PITR| is enabled, |pbm-agent| starts slicing the :term:`oplog <Oplog>`. The |pbm-agent| to do the slicing is elected in the same way as secondary nodes elect a new primary. (Find more information in :ref:`pbm.architecture.agent`.)
+
+A slice covers a 10 minute span of oplog events. It can be shorter if |PITR| is disabled or if slicing is stopped by a backup operation.
+
+The oplog slices are stored in the :file:`pbmPitr` subdirectory in the :ref:`remote storage defined in the config <pbm.config.initialize>`. A slice name reflects the start and end time this slice covers. 
+
+The |pbm-list| output includes both backup snapshots and valid time ranges for recovery. It also shows the |PITR| status.
+
+.. code-block:: bash
+
+   $ pbm list
+
+   Backup snapshots:
+  	2020-07-10T12:19:10Z
+  	2020-07-14T10:44:44Z
+  	2020-07-14T14:26:20Z
+  	2020-07-17T16:46:59Z
+   PITR <on>:
+  	2020-07-14T14:26:40 - 2020-07-16T17:27:26
+  	2020-07-17T16:47:20 - 2020-07-17T16:57:55
+
+.. note::
+
+   If you just enabled |PITR|, the time range list in the ``pbm list`` output is empty. It requires 10 minutes for the first chunk to appear in the list.  
+
+Restore to the point in time
+============================
+
+.. important::
+
+   Since a restore operation breaks the time line of oplog slicing, you must disable |PITR| before restoring a database:
```

Comment:
It is not critical, a and I don't know if should mention that, but anyway: the remark that the restore breaks up the timeline is more related to the resumption of the PITR slicing (necessity of the backup after the restore). PITR slicing and restore are not compatible just because the PITR slicer during the restore would capture restore events and put it into the "backup" which is illogical and utterly useless (if not dangerous to the data integrity).

Reply:
So this should say:
"Since a restore operation breaks the time line of oplog updates |PITR| will be automatically disabled as the restore starts."
right @dAdAbird ?

Reply:
As far as I understand PITR must be disabled manually. It's not done automatically.

Reply:
Yes, it should be disabled manually, no automation here. My comment was rather a terminology nitpicking.

Reply:
OK, so it should still be disabled manually. So the instructions just below this line are correct.